wordle-web/src/views/AboutView.vue

```diff
@@ -1,25 +1,13 @@
 <template>
-  <v-sheet color="blue" height="170px" class="pt-10 my-5">
-    <v-card color="indigo-darken-3" class="ma-5" elevation="5" rounded="xl">
-      <v-card-title>Hello World</v-card-title>
-    </v-card>
-  </v-sheet>
-
-  <v-row no-gutters>
-    <v-col v-for="i in 10" :key="i" cols="12" sm="6" md="4" lg="3">
-      <v-card class="ma-1">
-        <v-card-title> {{ i }} </v-card-title>
-      </v-card>
-    </v-col>
-  </v-row>
+  <v-card>
+    <v-card-item class="justify-end"
+      ><v-btn dark variant="outlined" to="/"> Home </v-btn></v-card-item
```

Comment:
Idk if it's important but in the instructions it should be labeled go back.
```suggestion
      ><v-btn dark variant="outlined" to="/"> Back </v-btn></v-card-item
```

Reply:
Home is more descriptive. Back doesn't work since you might have been on a page other than the home page before you navigated to the current page.